### ISSUE TITLE
Train the final layer

### ISSUE THREAD

**Issue by monjurulkarim** [feiyuhuahuo/Yolact_minimal#7]
Hi,
I have a very small dataset. I would like to train only the final layer after freezing all other layers. How can I do that? Any help is highly appreciated.

**Comment by feiyuhuahuo** [comment]
You can use `.detach()` to cut off the gradients. 
E.g. I can add .detach() here, and then only the final  self.conf_layer, self.bbox_layer and self.mask_layer would be trained.
![image](https://user-images.githubusercontent.com/32631344/74410340-d336a200-4e73-11ea-94c2-26800528736d.png)

**Comment by monjurulkarim** [comment]
@feiyuhuahuo Thank you for your response. I'll try as per your method.
